core/eval_test.go

```diff
@@ -1,30 +1,87 @@
 package core
 
 import (
+	"bytes"
 	"errors"
 	"strconv"
+	"strings"
 	"testing"
 	"time"
+	"unsafe"
 
+	"github.com/bytedance/sonic"
 	"gotest.tools/v3/assert"
 )
 
 type testCase struct {
-	input  []string
-	output []byte
+	setup     func()
+	input     []string
+	output    []byte
+	validator func(output []byte)
+}
+
+func setupTest() {
+	store = make(map[unsafe.Pointer]*Obj)
+	keypool = make(map[string]unsafe.Pointer)
+	expires = make(map[*Obj]uint64)
```

Comment:
You can call `RestetStore()` here instead.
I also noticed that `ResetStore()` doesn't reset the KeyspaceStat map, so please add the line below into the ResetStore function as well as a part of this PR.